openstack/compute/v2/extensions/floatingip/requests.go

```diff
@@ -79,7 +112,33 @@ func Associate(client *gophercloud.ServiceClient, serverId, fip string) Associat
 	return res
 }
 
+// AssociateFloatingIP pairs an allocated floating IP with an instance.
+func AssociateFloatingIP(client *gophercloud.ServiceClient, opts AssociateOpts) AssociateResult {
```

Comment:
I agree. I debated this when I originally wrote it and decided to wait and see if someone else flagged it :)